pkg/controller/operators/olm/operator.go

```diff
@@ -361,16 +415,26 @@ func (a *Operator) removeDanglingChildCSVs(csv *v1alpha1.ClusterServiceVersion)
 	operatorNamespace, ok := csv.Annotations[v1alpha2.OperatorGroupNamespaceAnnotationKey]
 	if !ok {
 		logger.Debug("missing operator namespace annotation on copied CSV")
+		// TODO: Should we clean up the CSV if we don't know where its parent is?
 		return fmt.Errorf("missing operator namespace annotation on copied CSV")
 	}
 
-	_, err := a.lister.OperatorsV1alpha1().ClusterServiceVersionLister().ClusterServiceVersions(operatorNamespace).Get(csv.GetName())
+	delete := false
+	parent, err := a.lister.OperatorsV1alpha1().ClusterServiceVersionLister().ClusterServiceVersions(operatorNamespace).Get(csv.GetName())
 	if k8serrors.IsNotFound(err) || k8serrors.IsGone(err) {
-		logger.Debug("deleting CSV since parent is missing")
+		logger.Debug("deleting copied CSV since parent is missing")
+		delete = true
+	} else if parent != nil && parent.Status.Phase == v1alpha1.CSVPhaseFailed && parent.Status.Reason == v1alpha1.CSVReasonInterOperatorGroupOwnerConflict {
+		logger.Debug("deleting copied CSV since parent has intersecting operatorgroup conflict")
```

Comment:
We remove copied CSVs in intersecting namespaces whose owners are members of OperatorGroup's that did not win the intersection.